### PR TITLE
k8s operator支持

### PR DESCRIPTION
**Pull request by gaoyan1998**
1. 增加对k8s operator提交方式支持
2. 取消使用flink-k8s-operator-api引入，仅重写部分关键代码负责提交功能即可，可以支持java8与java11

### CODE REVIEW

**Comment by aiwenmo** [comment]
It's a good feature.

File: dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java [hunk 1]
```diff
@@ -1048,9 +1051,12 @@ public JobInstance refreshJobInstance(Integer id, boolean isCoercive) {
...
             history.setConfig(JSONUtil.parseObject(history.getConfigJson()));
             if (Asserts.isNotNull(history.getClusterConfigurationId())) {
-                jobInfoDetail.setClusterConfiguration(
+                ClusterConfiguration clusterConfigById =
                         clusterConfigurationService.getClusterConfigById(
-                                history.getClusterConfigurationId()));
+                                history.getClusterConfigurationId());
+                jobInfoDetail.setClusterConfiguration(clusterConfigById);
+                Task task = getById(jobInstance.getTaskId());
+                jobInfoDetail.getInstance().setType(task.getType());
```
Comment (aiwenmo) [inline dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java]: Why set type to instance?

Comment (ikiler) [reply dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java:1059]: in handleJobDone function
at line 1428
`jobConfig.getGatewayConfig().setType(GatewayType.get(jobInstance.getType()));`
buildGatewayConfig need task type
we need to delete flink container when job has finished

Comment (gaoyan1998) [reply dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java]: > Why set type to instance?

in handleJobDone function
at line 1428
`jobConfig.getGatewayConfig().setType(GatewayType.get(jobInstance.getType()));`
buildGatewayConfig need task type
we need to delete flink container when job has finished

Comment (aiwenmo) [reply dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java]: history.getType()

**Review by aiwenmo — APPROVED** [review]
LGTM

**Pull request merged**